src/sentry/api/endpoints/project_group_index.py

```diff
@@ -450,7 +450,9 @@ def put(self, request, project):
             # a bit too complicated right now
             query_kwargs['limit'] = 1000
```

Comment:
Maybe assign this to a local variable `limit` (e.g. `limit = query_kwargs['limit'] = 1000`) and use it below too?